templates/home.tmpl

```diff
@@ -1,158 +1,72 @@
 #? stdtmpl | standard
 #proc generate_search_box(value=""): string =
 #  result = ""
-<div class="columns">
-  <div class="column col-12 search-box">
-    <form action="/search" method="GET" role="search" class="input-group">
-      <input class="form-input" name="query" placeholder="Search Nimble packages" type="search" value="$value" minlength="1" autofocus required >
-      <button type="submit" class="btn btn-primary input-group-btn">
-        <i class="fa fa-search"></i> Search
-      </button>
-    </form>
-  </div>
-</div>
+<form action="/search" method="GET" role="search" id="searchbox">
+  <input id="search" name="query" placeholder="Search Nimble packages" type="search" value="$value" minlength="1" autofocus required >
+</form>
 #end
 #
 #proc generate_home_page(most_queried_packages, new_packages: seq[Pkg], github_trending: seq[JsonNode]): string =
 #  result = ""
-
-${generate_search_box()}
-
-<div class="columns">
-  <div class="column col-12">
-    <p><b>Developer?</b> Search for packages or jump directly to a package page.
+<header class="hero">
+    <img class="logo" src="img/nimble-logo.png" alt="logo">
+    <h1>Nimble</h1>
+    <h5 style="text-align: center;">Nimble Package Directory</h5>
+    ${generate_search_box()}
+    <p>You don't have Nim in your computer. Install it here.</p>
```

Comment:
This is false.

Reply:
What did you mean false? The search box or you want to remove the install button?

Reply:
"You don't have Nim in your computer. "

Reply:
Ok, I have already removed that so this is resolved.